edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java

```diff
@@ -369,13 +384,28 @@ private Boolean checkIfBlock(ASTNode node) {
 						|| condition.contains("DEBUG") || condition.contains("TRACE")) {
 					LOGGER.info("We meet a logging wrapping: \n" + node);
 					return true;
-				}
+				} else
+                                    return false;
```

Comment:
@saledouble I don't think we want to change this:
1. We are trying to avoid big changes at this point.
1. We want to keep going up to see other `if` statements to see if there is wrapping.

Reply:
I see.